api/passwordreset.go

```diff
@@ -0,0 +1,10 @@
+package api
+
+type PasswordResetRequest struct {
+	Email string `json:"email"`
+}
+
+type VerifiedResetPasswordRequest struct {
+	Token    string `json:"token"`
+	Password string `json:"password"`
```

Comment:
Should all of these be required fields (`validate.Required`) ?  
Should `email` use `validate.Email` ?

Any min/max length requirements for token and password?

Reply:
sounds good. Password is going to be more complex as it depends on the org's requirements. It'll be the first validator that requires db access to look up the requirements.

Reply:
Oh right! Our password validation happens at a different layer. I don't think we need to mix those things. I think it's fine if this does static validation, and we do more involved validation later on once we have access to the DB.